author/viewsauth.py

```diff
@@ -39,7 +40,24 @@ def post(self, request):
 # Token based
 class Login(APIView):
     """
-    Handles a GET with a Basic Auth and passes back a token.
+    Handles a GET with a Basic Auth and passes back a token and profile information.
+
+    Response format:
+
+    {
+      "profile": {
+          "user": {
```

Comment:
That hasn't been added to the model yet... It should automatically pick it up if the field is added to serializer.